styles.css

```diff
@@ -0,0 +1,733 @@
+:root{
+    --primary-color:  #374151 ;
+    --dark-pink: #e70161;
+    --border-color: rgb(146, 146, 146);
+
+}
+body
+{
+    margin: 0rem;
+    padding: 0rem;
+}
+.a-nav2
+{
+    text-decoration: none;
+    color: var(--dark-pink);
+}
+.navigation
+{
+    background-color: var(--primary-color);
+    color:rgb(250, 250, 255);
+    height: 4rem;
+    border-radius: 1rem;
+}
+.container
+{
+    padding: 0rem 1rem;
+}
+.navigation .nav-developer
+{
+font-weight: bold;
+padding: 1rem;
+font-size: 25px;
+display: inline;
+}
+.list-non-bullet
+{
+    list-style: none;
+    padding-inline-start: 800px;
+   padding-block-start: 1rem;
+    font-size: 22px;
+    display: inline;
+}
+.list-item-inline
+{
+    display: inline;
+    padding: 0rem 0.5rem
+}
+.navigation .nav-pills
+{
+    text-align: right;
+}
+.navigation .link
+{
+    color: white;
+    text-decoration: none;
+}
+.navigation .link-active
+{
+    font-weight: bold;
+}
+.list-non-bullet-nav2
+
+{
+    list-style: none;
+}
+.nav2{
+    display: flex;
+}
+.nav2-li
+{
+    padding: 1rem;
+    font-size: 20px;
+    color: var(--dark-pink);
+    
+}
+.nav2-li:hover{
+    cursor: pointer;
+    background: rgb(202, 200, 200);
+    border-radius: 5%;
+    width:80px;
+}
+.sec
+{
+    display: flex;
+    flex-direction: row;
+}
+.main-div
+{
+    display: flex;
+}
+.avatar-large
+{
+    width: 90px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.avatar-medium
+{
+    width: 70px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.avatar-small
+{
+    width: 50px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.avatar-extra-small
+{
+    width: 30px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.img-div
+{
+    width: 750px;
+    background-color: black;
+}
+.div-h2
+{
+    color: var(--primary-color);
+    font-size: 28px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.sub-div{
+padding: 0.5rem 0rem 0rem 8.5rem;
+
+}
+.div2
+{
+    padding: 0.5rem 0rem 0rem 8.5rem;
+}
+.alert1
```

Comment:
`border` ,`border-radius`,` font-size`,`width`,`height`,`padding` & `color` are common in alert1, alert2, alert3 . You can put it in a separate parent alert class and then use it so the code looks cleaner and wont repeat